js/navigation.js

```diff
@@ -238,6 +242,18 @@ bcpl.navigation = (($, keyCodes) => {
 		}
 	};
 
+	const toggleMobileNavCallout = () => {
+		const isSubMenuVisible = $(mobileNavSubmenuItemSelector).hasClass('active');
+		const $mobileNavCallout = $(mobileNavCalloutSelector);
+
+		if (isSubMenuVisible) {
+			$mobileNavCallout.show();
+		} else {
+			$mobileNavCallout.show();
+		}
```

Comment:
Lines 250 and 252 are doing the same thing. Oops?

Reply:
oh crap...wth...thanks, good catch

Reply:
@mxsnyder ready for another look. I actually wasn't even using that method. So it has been removed